addons/account_asset/account_asset.py

```diff
@@ -400,51 +407,92 @@ def _get_move_check(self):
         self.move_check = bool(self.move_id)
 
     @api.multi
-    def create_move(self, post_move=True):
+    def create_move(self, post_move=True, group_entries=False):
         created_moves = self.env['account.move']
-        for line in self:
```

Comment:
this is not elegant: I'd suggest to create a function that receive all the parameters (amount, credit account, debit account...) that will be called by both the cases where group_entries is True or False in order to reduce duplicated code.